docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
+Elastic Uptime allows you to monitor the availability and response times of applications and services in real time and to detect problems before they affect users.
+
+Elastic Uptime is based on {heartbeat-ref}/index.html[Heartbeat], an open source uptime monitor.
+Elastic Uptime is transparent, flexible, and developer-friendly.
+It can help you to understand uptime and response time characteristics for your services and applications.
+It can be deployed both inside and outside your organization's network, so you can analyze problems from multiple vantage points.
+
+Elastic Uptime consists of these components: *Heartbeat*, *Elasticsearch* and *Kibana*.
+
+[float]
+=== Heartbeat
+
+{heartbeat-ref}/index.html[Heartbeat] is an open source data shipper that performs uptime monitoring.
+Heartbeat allows you to collect monitoring data from your target applications and services, and ship it to Elasticsearch.
+
+[float]
+=== Elasticsearch
+
+{ref}/index.html[Elasticsearch] is a highly scalable, open source, search and analytics engine.
+Elasticsearch can store, search, and analyze large volumes of data in near real-time.
+Elastic Uptime uses Elasticsearch to store monitoring data from Heartbeat in Elasticsearch documents.
+
+[float]
+=== Kibana
+
+{kibana-ref}/index.html[Kibana] is an open source analytics and visualization platform designed to work with Elasticsearch.
+You can use Kibana to search, view, and interact with data stored in Elasticsearch.
+You can easily perform advanced data analysis and visualize your data in a variety of charts, tables, and maps.
+
+The {kibana-ref}/xpack-uptime.html[Elasticsearch Uptime app] in Kibana provides a dedicated user interface for viewing uptime data and identifying problem areas.
+
+[float]
+=== Example deployments
+// ++ I like the Infra/logging diagram which shows Infrastructure and Logging apps as separate components inside Kibana
+// ++ In diagram, should be Uptime app, not Uptime UI, possibly even Elastic Uptime? Also applies to Infra/logging/APM.
```

Comment:
I think this is a pretty subjective call. I think UI is solid because it's unambiguous that it's the UI app inside of Kibana. Perhaps we could use "app" if we phrased it like "Uptime (Kibana) App"

Reply:
According to marketing guidelines (https://wiki.elastic.co/display/MAR/Term+usage+guide) it does need to be "Uptime app", and "app" can be dropped if it's clear from the context. The term "UI" is reserved for specific references to the actual user interface. 

With this in mind, I can't actually think of many instances where I'd even want to refer to the Uptime UI as a specific item. Even when writing about the UI, most of the words would be about components in the UI, not the UI as a whole.

Again, this is an area where we need to be consistent across all the Observability products, so these comments were really notes to me.